src/System.Security.Cryptography.X509Certificates/tests/CollectionImportTests.cs

```diff
@@ -309,6 +310,26 @@ public static void ImportPkcs12File_Chain_VerifyContents(X509KeyStorageFlags key
             }
         }
 
+#if netcoreapp11
+        [Fact]
+        [PlatformSpecific(TestPlatforms.OSX)]
+        public static void EphemeralKeySet_OSX()
+        {
+            // EphemeralKeySet fails when loading a PFX, and is ignored otherwise.
+            using (ImportedCollection coll = Cert.Import(TestData.Pkcs7ChainDerBytes, null, X509KeyStorageFlags.EphemeralKeySet))
+            {
+                Assert.Equal(3, coll.Collection.Count);
+            }
+
+            Assert.Throws<PlatformNotSupportedException>(
+                () =>
+                {
+                    using (Cert.Import(TestData.EmptyPfx, null, X509KeyStorageFlags.EphemeralKeySet))
```

Comment:
Fair enough.  I'm pretty sure I was guarding it so that if it didn't throw it wouldn't finalize... but I guess I wouldn't care about the finalization numbers when tests were failing, and there's prior art for asserting that the X509Certificate2 ctor throws without having a using.